src/routes/search/handlers.js

```diff
@@ -0,0 +1,25 @@
+import { search } from '../../lib/awsElasticSearch';
+import handleErrors from '../../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:SITE_SEARCH';
+
+const logContext = {
+  namespace,
+};
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I know this is kind of a hypothetical, but I think in execution we probably want the index to be hardcoded and have different routes for searching different indices (to allow for further config instead of having to pass a bunch of stuff through the query)

Reply:
I could go either way. I liked the idea of passing an array of indexes to search (ie one api path). If no index names are passed we search all.

Probably warrants a larger conversation.